### PR TITLE
Created feed and post related JSON Schemas

### PR DESCRIPTION
**Pull request by redsolver**
Uses https://json-schema.org/
VS Code can automatically load, check, use and autocomplete JSON Schemas - open this repo with VS Code to try it out.

### CODE REVIEW

**Review by DaWe35 — DISMISSED** [review]
I don't think I can be constructive, Redsolver knows what he needs for social pages.

**Review by lukehmcc — DISMISSED** [review]
Yeah I agree with DaWe, I did some reading but I don't quite understand how it work
Though I trust red so I'ma just drop an approve

File: draft-01/post.schema.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://skystandards.hns.siasky.net/draft-01/post.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "A representation of a post",
+    "type": "object",
+    "required": [
+        "id",
+        "content"
```
Comment (ilkamo) [inline draft-01/post.schema.json]: > "repostOf": {
>             "$ref": "../draft-01/postReference.schema.json",
>             "description": "Full ID of the post being reposted (If this key is present, this post is a repost and does not need to contain a \"content\")"
>         }

Are we sure that `content` is required? What if `repostOf ` is present?

Comment (redsolver) [reply draft-01/post.schema.json]: Thanks, removed it!

**Review by ilkamo — APPROVED** [review]

**Review by lukehmcc — APPROVED** [review]

**Pull request merged**